ci/cloudbuild/builds/cmake-split-install.sh

```diff
@@ -38,24 +38,24 @@ install_args=(
   # Test with shared libraries because the problems are more obvious with them.
   -DBUILD_SHARED_LIBS=ON
 )
-io::log_h2 "Building and installing core libraries"
+
+io::log_h2 "Building and installing common libraries"
 # We need a custom build directory
+mapfile -t core_cmake_args < <(cmake::common_args cmake-out/common-libraries)
+io::run cmake "${core_cmake_args[@]}" "${install_args[@]}" \
+  -DGOOGLE_CLOUD_CPP_ENABLE="__common__"
+io::run cmake --build cmake-out/common-libraries
+io::run cmake --install cmake-out/common-libraries --prefix "${INSTALL_PREFIX}"
+
+io::log_h2 "Building and installing popular libraries"
```

Comment:
Given the names of the "args" array, and the cmake-out directory, perhaps we should `s/popular/core/` for consistency.